backend/model/books.py

```diff
@@ -35,11 +35,17 @@ def __init__(self, isbn, stock, precio, titulo, autor=None, editorial=None, sino
         else:
             self.fecha_de_publicacion = fecha_de_publicacion
 
-    def json(self):
+    def json(self, reviews=False, score=False):
         _ignore = self.isbn  # Forces execution to parse properly the class, fixing the bug of transient data
         atr = self.__dict__.copy()
         del atr["_sa_instance_state"]
-        atr['fecha_de_publicacion'] = self.fecha_de_publicacion.strftime('%d-%m-%Y')
+        atr['fecha_de_publicacion'] = self.fecha_de_publicacion.strftime('%Y-%m-%d')
+        if reviews:
+            reviews = self.reviews
+            atr['reviews'] = [review.json() for review in reviews] if reviews else []
```

Comment:
No es neccesari la comprovacio del if crec que sino no hi ha cap element a la relacio retorna una llista buida.
```suggestion
            atr['reviews'] = [review.json() for review in self.reviews]
```